C7GameData/ImportCiv3.cs

```diff
@@ -63,6 +63,7 @@ public static C7SaveFormat ImportSav(string savePath, string defaultBicPath)
                 c7Tile.riverSouthwest = civ3Tile.RiverSouthwest;
                 c7Tile.riverNorthwest = civ3Tile.RiverNorthwest;
                 c7Tile.Resource = resourcesByIndex[civ3Tile.ResourceID];
+				c7Tile.ResourceKey = resourcesByIndex[civ3Tile.ResourceID].Key;
```

Comment:
This line doesn't match the others on my screen due to tabs vs spaces. We really ought to go through and tabify everything once and for all.

Reply:
Spaces vs tabs :(.  I have now standardized the rest of this file on tabs.  I wonder if there's a good way to do it in one fell swoop without manually replacing them?  I could write a program to do that, but there's probably already a good way.

Reply:
I don't know of an easy way to do it. I thought we only needed to change a handful of files, but searching the repo for strings of 8 spaces turned up almost 100 matching files, including all of the QueryCiv3 module.

Reply:
Yeah... there was no consistency on that early on.  It didn't help that when I thought I'd solved it in my VSCode settings, so all new code would be tabs, I didn't realize that the setting was only applied to the file I had been editing, so it continued to add spaces in other files for a while.

It looks like there's a somewhat okay to do this in VSCode (https://stackoverflow.com/questions/33974681/how-can-i-convert-tabs-to-spaces-and-vice-versa-in-an-existing-file).  I'm thinking we should have a "Carthage" branch off of Development, for any intermediate code that we do before Babylon releases (which won't be merged before then).  That could then be a good place to have one big commit to convert them.

I'm a little bit afraid of doing it before you merge your current PR though, in case it causes conflicts all over the place.